node/app/Repo.ts

```diff
@@ -33,11 +40,10 @@ export namespace Repo {
 	): Promise<number> {
 		const revWalk = repo.createRevWalk();
 		revWalk.push(before.id());
-		let i = 1;
+		let i = 0;
 		/// ^^ Include `before` in the count.
 		while (true) {
 			try {
-				await revWalk.next();
```

Comment:
Huuuuuuh, sure? 😬

Reply:
(I think there is now an infinite `while` no?)

Reply:
Weird! Not sure what happened here.